codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpProtocolServiceClient.kt

```diff
@@ -32,6 +32,14 @@ class AWSHttpProtocolServiceClient(
     private val writer: SwiftWriter,
     private val serviceConfig: ServiceConfig,
 ) : HttpProtocolServiceClient(ctx, writer, serviceConfig) {
+    // Enabling the following code will cause the SDK to get version from AWSClientRuntime
+//
+//    override var clientProtocolSymbol = AWSClientRuntimeTypes.Core.AWSClient
+//
+//    override fun renderVersionProperty() {
+//        // No content, version property is provided by a protocol extension on AWSClient
+//    }
```

Comment:
The code above will be used once the two new CLI subcommands are running in the build system.